x-pack/plugins/index_lifecycle_management/public/application/sections/policy_table/policy_table.tsx

```diff
@@ -119,67 +122,61 @@ export const PolicyTable: React.FunctionComponent<Props> = ({
     );
   } else {
     return (
-      <EuiPageBody>
-        <EuiPageContent verticalPosition="center" horizontalPosition="center">
-          <EuiEmptyPrompt
-            iconType="managementApp"
-            title={
-              <h1>
+      <ManagementPageLayout isEmptyState={true}>
```

Comment:
**Step 3:** For every rendered view, use the `useKibana` hook to access `managementPageLayout`, cast it to `ManagementPageLayout` so it looks like a React component, and wrap the rendered view in it. Here on line 125 we're migrating an empty rendered view. On line 153, below, we're migrating a non-empty rendered view. Note the differences in the deltas.